linux/just_files/docker_functions.bsh

```diff
@@ -754,6 +793,125 @@ function docker_compose_volumes()
   done
 }
 
+#**
+# :Arguments: * ``$1`` - Name of service of interest
+#             * [``$2``] - Prefix to volume names
+# :Input: **stdin** - Yarped docker compose yaml file
+# :Output: * ``DOCKER_VOLUME_SOURCES`` - List of sources
+#          * ``DOCKER_VOLUME_TARGETS`` - Corresponding list of targets
+#          * ``DOCKER_VOLUME_FLAGS`` - Corresponding list of flags. Can be a mix of short and long format flags. Needs more parsing before this mix is useful.
```

Comment:
what does "Needs more parsing before this mix is useful" mean

Reply:
bad copy and paste, will be replaced